core/templates/dev/head/pages/collection_editor/editor_tab/collection_node_creator_directive.html

```diff
@@ -5,6 +5,6 @@
 </form>
 or
 <form ng-submit="addExploration()">
-  <input ng-model="newExplorationId" type="text" placeholder="New exploration ID">
-  <button type="submit" ng-disabled="!newExplorationId">Add Existing Exploration</button>
+  <input class="form-inline" ng-model="newExplorationId" type="text" placeholder="New exploration ID">
```

Comment:
@seanlip  - I'll work on getting the new commits to my branch tonight. Was running into some issues last night. 
Just noticed this as well. It says "New Exploration ID" when we are adding an existing exploration. Should we change it to just "Exploration ID"?